deploy/terraform-datacommons-website/modules/apikeys/main.tf

```diff
@@ -45,7 +45,8 @@ resource "google_secret_manager_secret" "maps_api_key_secret" {
   replication {
     user_managed {
       replicas {
-        location = var.location
+        # location needs to be a region here, so if var.location is a zone, make it a region.
+        location = length(split(var.location, "-")) == 2 ? var.location : join("-", slice(split( "-", var.location), 0, 2))
```

Comment:
wonder if there are tf test to ensure these expressions can be changed later?

Reply:
The closest thing I found was the Terraform console, for interactive expression testing. As for changing the expression, we'll likely have to test it on a live resource (which we will have to do anyways when we release)

https://developer.hashicorp.com/terraform/cli/commands/console

Reply:
Then an e2e tests would be important